client/state/comments/test/fixtures/nock-requests.json

```diff
@@ -0,0 +1 @@
x-prefork":"1","x-hacker":"Oh, Awesome: Opossum","expires":"Wed, 11 Jan 1984 05:00:00 GMT","cache-control":"no-cache, must-revalidate, max-age=0","pragma":"no-cache","content-encoding":"gzip","x-ac":"2.ams _dfw","strict-transport-security":"max-age=15552000"}},{"scope":"https://public-api.wordpress.com:443","method":"GET","path":"/rest/v1.1/batch?urls%5B%5D=%2Fsites%2F106350587%2Fcomments%2F13","body":"","status":200,"response":["1f8b0800000000000003b554db8e9b3010fd95d455f7a5106cb381046d765f5a5595ba2f55fb500929726048dce562d9266976b5ffbe632057f5ad2912923d973373660ebc903430d282490346a3704227d3380db2a6aaa0b6ce1892e4857cfd4412167a4435c6eeefdc2356da1248421e7723b105d354307211ef08ba76ca79ba048f94b27ec2dbda5a6592344803d52e4b99f942c9f1b6d1b9d260cc18aba6019e6c1a6cd898fdad318787164e5e3d225abb6ef4a13d16c6533e89b158b3923556dbb57a57cb3f581e2a214b9214a234e0915a54aeb55fe81e3dca6cdd6a0cf74821b5b18b13a7eb5b1c4d27a1b5cc606f3e56f9f9fddbc0b1a3f85b6c84c9b454366fecb27d7e3e678a091860855e1cf3fad9d0f14af7ae7e22fd390d0a56f0db493c5df22c9fe5515c640cf8328e4114398b8ad98399cfa29b7c2e735c9dcc9afa46cfbf6019a59b4296b0b8e80fea8b3a47266eee8b6ea894ddce38e3118e3b17d6f1e594453ee53e9dfc602c617142c38f9426949e4da067e2107183f672c70e0379f2fead76fea01edfadf7a03e5f8915f82883f77b038a119bc3a5db4b325b35ae00e51167f039f4f9071e9ee5e0302c5e30e34eddafa12c1b6f948b873bccb84fdd468d15b635e8170ac7b581dccd4de82e6790cda0e801b713f5132cb2a67531d42372e10c87f00aac70da74d237ee60a02caef20d9c7d9c1e413aeaeab869d0c1f652b8023ad9ff3bfe19e9e41fe0110daa9460ae027b4e7f404e8361d1ffa346878b155ef1790305a32a7e88050000"],"headers":{"server":"nginx","date":"Tue, 09 Feb 2016 17:54:09 GMT","content-type":"application/json","transfer-encoding":"chunked","connection":"close","vary":"Accept-Encoding","x-prefork":"1","x-hacker":"Oh, Awesome: Opossum","expires":"Wed, 11 Jan 1984 05:00:00 GMT","cache-control":"no-cache, must-revalidate, max-age=0","pragma":"no-cache","content-encoding":"gzip","x-ac":"2.ams _dfw","strict-transport-security":"max-age=15552000"}},{"scope":"https://public-api.wordpress.com:443","method":"GET","path":"/rest/v1.1/sites/106350587/posts/2/replies/?fields=ID%2Cdate&number=100","body":"","status":200,"response":["1f8b08000000000000037dd0b10e82400c80e177e9ea91b43dee0e3abbb8b319638c60e2000cc04478774f310ee5706cfaa569fe191efdd4d520440686e7d85c4fc738a0b70e5d110cdcfbb66dba710039cff0d95903f56d6c4080917c869ca1ab888482a03d200a222ce68b398531085b61d698fe60d418773096e248e13269bdb878bb54b6d8b7e895f55bcbd5bb43113329ebf62c6fb32512471b7ff0c241d944e1d5e662f5bf89c0ab65c97f1d2ecb0bf75fb4ce15020000"],"headers":{"server":"nginx","date":"Tue, 09 Feb 2016 17:54:10 GMT","content-type":"application/json","transfer-encoding":"chunked","connection":"close","vary":"Accept-Encoding","x-prefork":"1","x-hacker":"Oh, Awesome: Opossum","expires":"Wed, 11 Jan 1984 05:00:00 GMT","cache-control":"no-cache, must-revalidate, max-age=0","pragma":"no-cache","content-encoding":"gzip","x-ac":"2.ams _dfw","strict-transport-security":"max-age=15552000"}}]
```

Comment:
Just to clarify, You are bothered with the approach of recording the API responses
or you're bothered with the response being stored as hex bytes string "1f8b0800000000000003ed9d0b93dbb6b5c7b..." ?
( that's just gzip, note the magic `0x1f8b` )

I can actually not store anything (delete that file), but that'll slow the test(s) unnecessary...